doc/classes/PathFollow3D.xml

```diff
@@ -20,7 +20,7 @@
 		</method>
 	</methods>
 	<members>
-		<member name="cubic_interp" type="bool" setter="set_cubic_interpolation" getter="get_cubic_interpolation" default="true">
+		<member name="cubic_interpolation" type="bool" setter="set_cubic_interpolation" getter="is_cubic_interpolation" default="true">
 			If [code]true[/code], the position between two cached points is interpolated cubically, and linearly otherwise.
```

Comment:
Given the description, it sounds like there is always interpolation. So maybe it should be `is_interpolation_cubic`?

Reply:
Is `set_use_interpolation_cubic` / `is_using_interpolation_cubic` better?